src/events/payload.rs

```diff
@@ -109,6 +109,9 @@ pub enum EventType {
 
     /// A webxdc wants an info message or a changed summary to be notified.
     IncomingWebxdcNotify {
+        /// ID of the chat which the message belongs to.
```

Comment:
Again, it should be clarified what is "the message" because an info message is optional